src/components/ui/Button/button.module.scss

```
@@ -1,121 +1,121 @@
 @import "src/styles/scaffold";
 
-[data-fs-button] {
+.fs-button[data-fs-button] {
```

No need for the `data-attribute` here! See `product-card.module.scss` as reference.
```suggestion
.fs-button {
```